post-types/templates/page-profiles-list-page.php

```diff
@@ -10,7 +10,7 @@
     </div>
   </div>
   <?php
-    if (odm_language_manager()->get_current_language() !== 'en') {
+    if (odm_language_manager()->get_current_language() != 'en') {
```

Comment:
@Huyeng why you removed the '=' character? It is better to compare strings with === or !== so also the type is checked...